apps/system/index.html

```diff
@@ -290,7 +290,9 @@
                 <div class="other-notifications"></div>
               </div>
             </div>
+          </div>
 
+          <div id="tray-footer-container" class="tray-footer">
             <div id="utility-tray-footer">
```

Comment:
let's make it `<div id="utility-tray-footer" class="animate-out">`
otherwise, at least on a flame, the first time I open the utility tray I can see the footer flash midway